resources/views/partials/content-page-saved-searches.blade.php

```diff
@@ -1,4 +1,4 @@
-<div class="align-right" hidden><button id="remove-all" class="button button--borderless button--destructive">@svg('delete', 'icon--delete', ['focusable' => 'false', 'aria-hidden' => 'true']) {{ __('Remove all', 'coop-library') }}​​&#8203;<span class="screen-reader-text">{{ __('saved searches', 'coop-library') }}</span></button></div>
+<div class="align-right" hidden><button id="remove-all" class="button button--borderless button--destructive">@svg('delete', 'icon--delete', ['focusable' => 'false', 'aria-hidden' => 'true']) {{ __('Remove all', 'coop-library') }}​​&nbsp<span class="screen-reader-text">{{ __('saved searches', 'coop-library') }}</span></button></div>
```

Comment:
@greatislander I tested with NVDA and `&nbsp` worked in providing a break in the label. `&#8203;` did not work.

Reply:
Does `&nbsp;` inside the `<span class="visually-hidden">` work?

Reply:
@jhung I am moving this into a separate issue.